pkg/schedule/checker/rule_checker.go

```diff
@@ -108,7 +107,7 @@ func NewRuleChecker(ctx context.Context, cluster sche.CheckerCluster, ruleManage
 
 // GetType returns RuleChecker's Type
 func (*RuleChecker) GetType() string {
-	return ruleCheckerName
+	return config.RuleCheckerName.String()
```

Comment:
After this PR, the type will be changed from xx-xx to xx_xx.

Reply:
https://github.com/tikv/pd/pull/8316/files#r1650478721